main/webapp/modules/core/styles/views/data-table-view.less

```diff
@@ -55,7 +55,53 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
   font-weight: bold;
   }
 
-.data-table-header {
+.viewpanel-rowrecord a {
+  background-color: @fill_primary;
+  margin: 0px 1px 0px 1px;
+  padding: 2px 5px 2px 5px;
+  .rounded_corners(4px);
+  }
+
+.viewpanel-pagesize a {
+  background-color: @fill_primary;
+  margin: 0px 2px 0px 2px;
+  padding: 2px 3px 2px 3px;
+  .rounded_corners(4px);
+  }
+
+div.viewpanel-pagesize span:first-of-type {
+  margin: 0px 0px 0px 20px;
+}
+
+.viewpanel-paging a {
+  background-color: @fill_primary;
+  margin: 0px 4px 0px 4px;
+  padding: 2px 10px 2px 10px;
+  .rounded_corners(4px);
+  }
+
+.viewpanel-paging span#viewpanel-paging-current {
+  margin: 0 10px 0 10px;
+  padding: 0px;
+  }
+
+.viewpanel-paging input#viewpanel-paging-current-input {
+  width: 60px;
+  margin: 0px;
+  padding: 0px;
+  height: 16px;
+  position: relative;
+  top: -2px;
+  }
+
+.data-header-table-container {
```

Comment:
We don't need this style for ` .data-header-table-container` either, as that container was also removed (unless you're using it somewhere?). I missed it earlier.

Reply:
@lisa761: thank you so much for pointing me your changes in my PR. I would have had a very hard time to find them by myself.

I'm now pretty good, and you, can you run it smooth? Have you tried the Up/Down Arrow Keys?

Regards,
   Antoine

Reply:
Yes, it's pretty great actually!